AssertionError: The given network_url *'ganache'* ...

**Describe the bug**
If I use the default config.ini with barge, it complains with the traceback below.

**To Reproduce**

1. Do steps 1-3 in developers.md (Install dependencies, Run the services, Set up contracts)
2. Then open a python console and try the following. The last line will fail.

```python
import os
from ocean_lib.config import Config
from ocean_lib.ocean.ocean import Ocean
from ocean_lib.web3_internal.wallet import Wallet

private_key = os.getenv('TEST_PRIVATE_KEY1')
config = Config('config.ini')
ocean = Ocean(config)
```

Note: all pytest tests _do_ pass.

**Expected behavior**
The last line not to fail. While all pytest tests still passing.

**Config.ini contents**
For reference, here is part of the config.ini file:
```text
[eth-network]

network = 'ganache'
artifacts.path = ...
```

**Traceback**
```console
>>> ocean = Ocean(config)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/trentmc/code/ocean.py/ocean_lib/ocean/ocean.py", line 100, in __init__
    provider=get_web3_connection_provider(self._config.network_url)
  File "/home/trentmc/code/ocean.py/ocean_lib/ocean/util.py", line 79, in get_web3_connection_provider
    assert network_url in SUPPORTED_NETWORK_NAMES, (
AssertionError: The given network_url *'ganache'* does not start with either `http` or `wss`, in this case a network name is expected and must be one of the supported networks {'kovan', 'ropsten', 'mainnet', 'rinkeby', 'ganache'}.
```

--- a/config.ini
+++ b/config.ini
@@ -1,6 +1,6 @@
 [eth-network]
 
-network = 'ganache'
+network = ganache
 artifacts.path = ~/.ocean/ocean-contracts/artifacts
 address.file = ~/.ocean/ocean-contracts/artifacts/address.json
 

